packages/blazor-workspace/NimbleBlazor/README.md

```diff
@@ -0,0 +1,154 @@
+<div align="center">
```

Comment:
Realized we never had the README configured for the nuget. Can you add that so it's included? https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/

Reply:
Can you confirm you followed the referenced instructions or intentionally chose different settings?

Seems to indicate changes I don't see reflected here:
![image](https://github.com/ni/nimble/assets/1588923/a808a2ab-279a-4555-81ae-56e1ea232b3f)

Also has instructions for manual validation of the built nuget, did you try those?

![image](https://github.com/ni/nimble/assets/1588923/2df64b4c-454f-433f-8162-df7ae889fe20)

Reply:
I absolutely did _not_ follow the referenced instructions... 🤦 I don't know how I missed adding the first property. Now I've verified that it shows up properly when uploaded. When doing so, I noticed that the little HTML we had at the top wasn't rendering properly, so I removed it. I liked it, but it's probably not worth having if it's going to look bad on NuGet.org.